app/index.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app");
+const port = 3000;
```

Comment:
`process.env.PORT || 3000`

Reply:
I think that it should be implemented by person who has task with dotenv but okay, i'll do it as you wrote